cli/floatingip_set_rdns.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hetznercloud/hcloud-go/hcloud"
+	"github.com/spf13/cobra"
+)
+
+func newFloatingIPSetRdnsCommand(cli *CLI) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "set-rdns [FLAGS] FLOATINGIP",
+		Short:                 "Change reverse dns of a Floating IP",
+		Args:                  cobra.ExactArgs(1),
+		TraverseChildren:      true,
+		DisableFlagsInUseLine: true,
+		PreRunE:               cli.ensureToken,
+		RunE:                  cli.wrap(runFloatingIPSetRdns),
+	}
+
+	cmd.Flags().StringP("hostname", "r", "", "Primary IP address for which the reverse DNS entry should be set")
+	cmd.MarkFlagRequired("hostname")
+
+	cmd.Flags().StringP("ip", "i", "", "Hostname to set as a reverse DNS PTR entry")
+
+	return cmd
+}
+
+func runFloatingIPSetRdns(cli *CLI, cmd *cobra.Command, args []string) error {
```

Comment:
Name it `runFloatingIPSetRDNS`. In Go it’s unusual to use the `Http` instead of `HTTP` style seen in other languages.